src/engines/jade/gui/main/options.cpp

```diff
@@ -0,0 +1,182 @@
+/* xoreos - A reimplementation of BioWare's Aurora engine
+ *
+ * xoreos is the legal property of its developers, whose names
+ * can be found in the AUTHORS file distributed with this source
+ * distribution.
+ *
+ * xoreos is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * xoreos is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with xoreos. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *  The Jade Empire main options menu.
+ */
+
+#include "src/aurora/talkman.h"
+
+#include "src/engines/jade/gui/main/options.h"
+
+#include "src/engines/jade/gui/options/audio.h"
+#include "src/engines/jade/gui/options/video.h"
+#include "src/engines/jade/gui/options/diff.h"
+#include "src/engines/jade/gui/options/feed.h"
+#include "src/engines/jade/gui/options/control.h"
+
+#include "src/engines/kotor/gui/widgets/listbox.h"
+#include "src/engines/kotor/gui/widgets/label.h"
+
+namespace Engines {
+
+namespace Jade {
+
+OptionsMenu::OptionsMenu(Console *console) : GUI(console) {
+	load("options");
+
+	// Move this label behind the buttons.
+	float x, y, z;
+	getWidget("Lopt")->getPosition(x, y, z);
+	getWidget("Lopt")->setPosition(x, y, z + 10);
+
+	Engines::KotOR::WidgetListBox *optionsListBox = getListBox("OptionsListBox");
+
+	addWidget(optionsListBox->createItem("AUDIO_SETTINGS"));
+	addWidget(optionsListBox->createItem("GRAPHIC_SETTINGS"));
+	addWidget(optionsListBox->createItem("DIFFICULTY"));
+	addWidget(optionsListBox->createItem("GAME_INFO"));
+	addWidget(optionsListBox->createItem("CONTROLS"));
+	addWidget(optionsListBox->createItem("CREDITS"));
+
+	_audioOptionsButton = getButton("AUDIO_SETTINGS");
+	_videoOptionsButton = getButton("GRAPHIC_SETTINGS");
+	_difficultyOptionsButton = getButton("DIFFICULTY");
+	_gameInfoOptionsButton = getButton("GAME_INFO");
+	_controlOptionsButton = getButton("CONTROLS");
+	_creditsButton = getButton("CREDITS");
+
+	_backButton = getButton("ButtonBack");
+	_currentButton = 0;
+
+	_optionsDescription = getLabel("OptionsDescLabel");
+
+	_audioOptionsDescription = TalkMan.getString(129);
+	_videoOptionsDescription = TalkMan.getString(130);
+	_difficultyOptionsDescription = TalkMan.getString(152);
+	_gameInfoOptionsDescription = "";
```

Comment:
In English, that menu item is called "Feedback". The TLK string is 153.

Reply:
This are only the descriptions, the text which appears in the box below the options when hovering ofer the buttons. It was for the Feedback/gameinfo menu empty when hovering over it (at least in the german version).

Reply:
Well, that's a bug in the German version then. The English version does have a text there and it's 153.

Reply:
![untitled](https://user-images.githubusercontent.com/156130/35183945-9006e558-fdee-11e7-8151-be9f7e84b841.png)